insta/src/lib.rs

```diff
@@ -227,6 +227,7 @@
 //! # these are used by cargo insta test
 //! test:
 //!   # also set by INSTA_TEST_RUNNER
+//!   # cargo-nextest binary path can be explicitly set by INSTA_CARGO_NEXTEST_BIN
```

Comment:
I don't feel like it's the best place to add the documentation. But not sure where to put it otherwise